Fix field names, folder names, etc

jisusDev opened the pull request:
change the name of the files, to their respective usage, example screens has the name appbar_example_screen, comments in the exports, etc

jisusDev committed:
Fix-last-comments-pr

Gitignore actualizado para que no se comiteen los archivos autogenerados de flutter, archivo de barril para las screens y widgets, enum actualizado para el route y el listComponents_screen

jisusDev committed:
add-parameters-text-to-send_evidence_button-attach_evidence_button

jisusDev committed:
adjusts

jisusDev committed:
fix-names-folders

change the name of the files, to their respective usage, example screens has the name appbar_example_screen, comments in the exports, etc

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_create/incidence_create_screen.dart:
```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import 'package:buscampz_flutter/buscampz_rayito_flutter_package/components/widgets.dart';
+import 'package:buscampz_flutter/buscampz_rayito_flutter_package/config/colors/app_colors.dart';
+import 'package:buscampz_flutter/buscampz_rayito_flutter_package/components/elevated_button/content/ryt_attach_evidence_button.dart';
```
Colocar en un export de Barril

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_create/incidence_create_screen.dart:
```diff
@@ -0,0 +1,59 @@
+    return const SafeArea(
+      child: Column(
+        children: [
+          AppBarBuscampz(
```
Agregar un TODO para saber que esto lo tienen que cambiar luego cuando se integren las ramas

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_create/incidence_create_screen.dart:
```diff
@@ -0,0 +1,59 @@
+      child: Column(
+        children: [
+          AppBarBuscampz(
+            title: "Incidencias",
```
i18n

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_create/incidence_create_screen.dart:
```diff
@@ -0,0 +1,59 @@
+          SizedBox(
+            height: 20,
+          ),
+          RYTIncidentTextField(),
```
implementar, callbacks, placeholder, y provider.

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_create/incidence_create_screen.dart:
```diff
@@ -0,0 +1,59 @@
+          ),
+          RYTIncidentTextField(),
+          RYTDetailIncidentTextField(),
+          RYTAttachPhotoButton(
```
Callback de Boton

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_create/incidence_create_screen.dart:
```diff
@@ -0,0 +1,59 @@
+          ),
+          Center(
+            child: Text(
+              "Cuando se envíe esta incidencia, se le notificará a los representantes e institución. Debes ser lo más explícito posible para entender mejor el inconveniente.",
```
i18n

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_create/incidence_create_screen.dart:
```diff
@@ -0,0 +1,59 @@
+              "Cuando se envíe esta incidencia, se le notificará a los representantes e institución. Debes ser lo más explícito posible para entender mejor el inconveniente.",
+              textAlign: TextAlign.center,
+              style: TextStyle(
+                color: AppColors.greyDark
```
TODO: esto no debe ser asi, el color debe ser gray, y con una configuración del sistema saber si es lightt o dark

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_create/incidence_create_screen.dart:
```diff
@@ -0,0 +1,59 @@
+            ),
+          ),
+          Spacer(),
+          RYTSendEvidenceElevatedButton(
```
Callback de Boton

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_create/incidence_create_screen.dart:
```diff
@@ -0,0 +1,59 @@
+          ),
+          Spacer(),
+          RYTSendEvidenceElevatedButton(
+            text: "Enviar Evidencia",
```
i18n

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_view/incidence_view_screen.dart:
```diff
@@ -0,0 +1,70 @@
+import 'package:buscampz_flutter/buscampz_rayito_flutter_package/components/card_response_client/content/client_content_answer_card.dart';
+import 'package:buscampz_flutter/buscampz_rayito_flutter_package/components/widgets.dart';
+import 'package:buscampz_flutter/buscampz_rayito_flutter_package/config/colors/app_colors.dart';
```
export barril

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_view/incidence_view_screen.dart:
```diff
@@ -0,0 +1,70 @@
+import 'package:buscampz_flutter/buscampz_rayito_flutter_package/components/card_response_client/content/client_content_answer_card.dart';
+import 'package:buscampz_flutter/buscampz_rayito_flutter_package/components/widgets.dart';
+import 'package:buscampz_flutter/buscampz_rayito_flutter_package/config/colors/app_colors.dart';
+import 'package:flutter/material.dart';
```
organizar los imports

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_view/incidence_view_screen.dart:
```diff
@@ -0,0 +1,70 @@
+      child: Column(
+        children: [
+          AppBarBuscampz(
+            title: "Incidencias",
```
i18n

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_view/incidence_view_screen.dart:
```diff
@@ -0,0 +1,70 @@
+    return const SafeArea(
+      child: Column(
+        children: [
+          AppBarBuscampz(
```
TODO

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_view/incidence_view_screen.dart:
```diff
@@ -0,0 +1,70 @@
+          AppBarBuscampz(
+            title: "Incidencias",
+          ),
+          UserDataIncidence(
```
TODO

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_view/incidence_view_screen.dart:
```diff
@@ -0,0 +1,70 @@
+          UserDataIncidence(
+            image:
+                'https://doc.cerp.ideria.co/assets/images/image-a5238aed7050a0691758858b2569566d.jpg',
+            title: 'Freddy Rodríguez',
```
Provider

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_view/incidence_view_screen.dart:
```diff
@@ -0,0 +1,70 @@
+            address: 'Colegio San Sebastián de los Altos Campos.',
+            route: 'Ruta AB32-5',
+          ),
+          DetailIncidence(
```
TODO

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_view/incidence_view_screen.dart:
```diff
@@ -0,0 +1,70 @@
+            description:
+                'Hola a todos, tuve un inconveniente con la llanta, la estoy reparando y luego continúo con la ruta. Voy a llegar un poco más tarde de lo acordado. Pueden chequear la App para ver el recorrido que voy a estar haciendo.',
+          ),
+          ClientContentAnswerCard(
```
Falta Rayito

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_view/incidence_view_screen.dart:
```diff
@@ -0,0 +1,70 @@
+          ),
+          DetailIncidence(
+            description:
+                'Hola a todos, tuve un inconveniente con la llanta, la estoy reparando y luego continúo con la ruta. Voy a llegar un poco más tarde de lo acordado. Pueden chequear la App para ver el recorrido que voy a estar haciendo.',
```
Provider

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_view/incidence_view_screen.dart:
```diff
@@ -0,0 +1,70 @@
+            image: "assets/images/image_client_example.png",
+          ),
+          Divider(),
+          ClientContentAnswerCard(
```
Falta Rayito

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_view/incidence_view_screen.dart:
```diff
@@ -0,0 +1,70 @@
+            image: "assets/images/image_client_example.png",
+          ),
+          Divider(),
+          ClientContentAnswerCard(
```
Falta Rayito

venganza147 commented on lib/buscampz_app_flutter/screens/incidence_view/incidence_view_screen.dart:
```diff
@@ -0,0 +1,70 @@
+          Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              RYTButtonSendWithTextField(),
```
Callback del Button, y donde se le pasa el icon?

venganza147 commented on lib/buscampz_rayito_flutter_package/components/card_response_client/content/client_content_answer_card.dart:
```diff
@@ -0,0 +1,69 @@
+import 'package:buscampz_flutter/buscampz_rayito_flutter_package/config/colors/app_colors.dart';
+import 'package:buscampz_flutter/buscampz_rayito_flutter_package/config/text_style/app_text_style.dart';
+
+class ClientContentAnswerCard extends StatelessWidget {
```
Falta RYT

venganza147 commented on lib/buscampz_rayito_flutter_package/components/card_response_client/content/client_content_answer_card.dart:
```diff
@@ -0,0 +1,69 @@
+import 'package:buscampz_flutter/buscampz_rayito_flutter_package/config/text_style/app_text_style.dart';
+
+class ClientContentAnswerCard extends StatelessWidget {
+  final String clientName;
```
Todos los parámetros Opcionales. TODOSSSSSSSSSSSS

venganza147 commented on lib/buscampz_rayito_flutter_package/components/card_response_client/ryt_client_answer_card.dart:
```diff
@@ -20,7 +20,11 @@ class ClientAnswerCard extends StatelessWidget {
-      child: const ClientContentAnswerCard(),
+      child: const ClientContentAnswerCard(
+        clientName: "Selena Hernández",
+        text: "Ok señor Freddye, estoy atento para bajar y buscar a daniel",
```
Ningún Componente tiene textoooooooooooooooooo, si no viene nada se deja vacío.

Los componentes no usan i18n

venganza147 commented on lib/buscampz_rayito_flutter_package/components/elevated_button/content/ryt_attach_evidence_button.dart:
```diff
@@ -2,15 +2,22 @@ import 'package:buscampz_flutter/buscampz_rayito_flutter_package/config/colors/a
-class AttachPhotoButton extends StatelessWidget {
-  const AttachPhotoButton({
+class RYTAttachPhotoButton extends StatelessWidget {
+  final String text;
```
Opcionales

venganza147 commented on lib/buscampz_rayito_flutter_package/components/elevated_button/content/ryt_attach_evidence_button.dart:
```diff
@@ -2,15 +2,22 @@ import 'package:buscampz_flutter/buscampz_rayito_flutter_package/config/colors/a
 import 'package:buscampz_flutter/buscampz_rayito_flutter_package/config/text_style/app_text_style.dart';
 import 'package:flutter/material.dart';
```
Material siempre de primerooooooooo

venganza147 commented on lib/buscampz_rayito_flutter_package/components/elevated_button/content/ryt_attach_evidence_button.dart:
```diff
@@ -2,15 +2,22 @@ import 'package:buscampz_flutter/buscampz_rayito_flutter_package/config/colors/a
+  final Color color;
+  const RYTAttachPhotoButton({
     super.key,
+    required this.text,
```
Opcionales

venganza147 commented on lib/buscampz_rayito_flutter_package/components/elevated_button/content/ryt_send_evidence_elevated_button.dart:
```diff
@@ -1,10 +1,13 @@
-class SendEvidenceElevatedButton extends StatelessWidget {
-  const SendEvidenceElevatedButton({
+class RYTSendEvidenceElevatedButton extends StatelessWidget {
+  final String text;
```
opcional

venganza147 commented on lib/buscampz_rayito_flutter_package/components/elevated_button/ryt_evidence_elevated_button.dart:
```diff
@@ -17,16 +18,19 @@ class EvidenceElevatedButtons extends StatelessWidget {
-            child: AttachPhotoButton(),
+            child: RYTAttachPhotoButton(
+              color: AppColors.bgBotLight,
+              text: "Adjuntar Foto",
```
No existe texto en los componentes

venganza147 commented on lib/buscampz_rayito_flutter_package/components/elevated_button/ryt_evidence_elevated_button.dart:
```diff
@@ -17,16 +18,19 @@ class EvidenceElevatedButtons extends StatelessWidget {
         ),
-        SendEvidenceElevatedButton(),
+        RYTSendEvidenceElevatedButton(
+          text: "Enviar Incidencia",
```
No existe texto en los componentes

venganza147 commented on lib/buscampz_rayito_flutter_package/components/text_field_incidentes/content/detail_incident_text_form_content.dart:
```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+import 'package:buscampz_flutter/buscampz_rayito_flutter_package/config/colors/app_colors.dart';
+
+//Change name of the class Spanish to English
```
TODO

venganza147 commented on lib/buscampz_rayito_flutter_package/components/text_field_incidentes/content/detail_incident_text_form_content.dart:
```diff
@@ -0,0 +1,48 @@
+          )
+        ],
+      ),
+      child: TextFormField(
```
1 solo componente que sea un textfield RYTTextField

venganza147 commented on lib/buscampz_rayito_flutter_package/components/text_field_incidentes/content/detail_incident_text_form_content.dart:
```diff
@@ -0,0 +1,48 @@
+        decoration: InputDecoration(
+          filled: true,
+          fillColor: Colors.white,
+          hintText: "Escribir...",
```
Sin textosssssss

venganza147 commented on lib/buscampz_rayito_flutter_package/components/text_field_incidentes/content/incident_content_text_form.dart:
```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:buscampz_flutter/buscampz_rayito_flutter_package/config/colors/app_colors.dart';
+
+//Change name of the class Spanish to English
```
eliminar

venganza147 commented on lib/buscampz_rayito_flutter_package/components/text_field_incidentes/content/incident_content_text_form.dart:
```diff
@@ -0,0 +1,46 @@
+import 'package:buscampz_flutter/buscampz_rayito_flutter_package/config/colors/app_colors.dart';
+
+//Change name of the class Spanish to English
+class RYTIncidentContentTextField extends StatelessWidget {
```
Tener 1 solo textField para todo

venganza147 commented on lib/buscampz_rayito_flutter_package/components/text_field_incidentes/ryt_detail_incident_text_form.dart:
```diff
@@ -1,11 +1,11 @@
-class DetailIncidentTextField extends StatelessWidget {
-  const DetailIncidentTextField({super.key});
+class RYTDetailIncidentTextField extends StatelessWidget {
+  const RYTDetailIncidentTextField({super.key});
```
Tener un componente que se llame TextForm y sin textossss

venganza147 commented on lib/buscampz_rayito_flutter_package/components/textfield_with_button_send/content/button_send/ryt_button_send_content.dart:
```diff
@@ -1,8 +1,8 @@
 
-class ButtonSendContent extends StatelessWidget {
-  const ButtonSendContent({
+class RYTButtonSendContent extends StatelessWidget {
```
No he visto todos los botones, pero si es como el TextField debes tener 1 solo componente de botones.

venganza147 commented on lib/buscampz_rayito_flutter_package/components/textfield_with_button_send/content/textfield_send/ryt_send_content_text_field.dart:
```diff
@@ -1,7 +1,7 @@
 
-class SendContentTextField extends StatelessWidget {
-  const SendContentTextField({
+class RYTSendContentTextField extends StatelessWidget {
```
1 solo textField

venganza147 commented on lib/buscampz_rayito_flutter_package/components/textfield_with_button_send/ryt_button_send_with_text_field.dart:
```diff
@@ -0,0 +1,19 @@
+import 'package:buscampz_flutter/buscampz_rayito_flutter_package/components/textfield_with_button_send/content/textfield_send/ryt_send_content_text_field.dart';
+
+class RYTButtonSendWithTextField extends StatelessWidget {
+  const RYTButtonSendWithTextField({super.key});
```
Quiero ver como esto va a quedar, jajajaja

venganza147 commented on lib/buscampz_rayito_flutter_package/components/widgets.dart:
```diff
@@ -10,4 +10,25 @@ export 'package:buscampz_flutter/buscampz_rayito_flutter_package/components/card
 
 export 'package:buscampz_flutter/buscampz_rayito_flutter_package/components/incidence_section/detail_incidence.dart';
```
widget_exports

venganza147 commented on lib/buscampz_rayito_flutter_package/config/router/routers.dart:
```diff
@@ -1,22 +1,27 @@
+import 'package:buscampz_flutter/buscampz_app_flutter/screens/incidence_create/incidence_create_screen.dart';
```
organizar los imports

venganza147 commented on lib/buscampz_rayito_flutter_package/config/router/routers.dart:
```diff
@@ -1,22 +1,27 @@
+  incidenceCreateScreen("/incidenceCreateScreen"),
+  incidenceViewScreen("/incidenceViewScreen");
+
+  //textFieldIncidentes('/textFieldIncidentes'),
```
eliminar código comentado

venganza147 commented on lib/buscampz_rayito_flutter_package/example/example_screens/client_answer_card_example_screen.dart:
```diff
@@ -16,7 +16,7 @@ class ClientAnswerCardScreen extends StatelessWidget {
             children: [
               Center(
-                child: ClientAnswerCard(),
+                child: RYTClientAnswerCard(),
```
Pasarle la información para que se pinte.

venganza147 commented on lib/buscampz_rayito_flutter_package/example/example_screens/detail_incident_text_field_example_screen.dart:
```diff
@@ -14,7 +14,7 @@ class DetailIncidentTextFieldScreen extends StatelessWidget {
         child: Container(
           color: AppColors.primary,
-          child: const DetailIncidentTextField(),
+          child: const RYTDetailIncidentTextField(),
```
llenar con la información

venganza147 commented on lib/buscampz_rayito_flutter_package/example/example_screens/evidence_elevated_button_example_screen.dart:
```diff
@@ -11,7 +11,7 @@ class EvidenceElevatedButtonScreen extends StatelessWidget {
       ),
       body: const Center(
-        child: EvidenceElevatedButtons(),
+        child: RYTEvidenceElevatedButtons(),
```
Inyectar la información al button

venganza147 reviewed (commented):
Buenas suerte

jisusDev committed:
future_update_comments_pr

New editable textFormField, new editable Elevated Button, l18n beta implemented, comments updates, ordered imports, ordered exports, etc.

jisusDev committed:
Merge branch 'colector' into future-fix-names-folders

# Conflicts:
#	lib/buscampz_rayito_flutter_package/components/buttom_sheet/group_buttons_bottom_sheet.dart
#	lib/buscampz_rayito_flutter_package/components/buttom_sheet/ryt_bottom_sheet.dart
#	lib/buscampz_rayito_flutter_package/components/text_tittles_incidents/ryt_incident_text_field.dart
#	lib/buscampz_rayito_flutter_package/components/widgets/card_response_client/content/client_content_answer_card.dart
#	lib/buscampz_rayito_flutter_package/components/widgets/elevatedButtonEvidencia/content/attach_evidence_button.dart
#	lib/buscampz_rayito_flutter_package/components/widgets/elevatedButtonEvidencia/content/send_evidence_elevated_button.dart
#	lib/buscampz_rayito_flutter_package/components/widgets/text_field_incidentes/detail_incident_text_form.dart
#	lib/buscampz_rayito_flutter_package/config/router/routers.dart
#	lib/buscampz_rayito_flutter_package/example/example_screens/card_incidence_list_screen.dart
#	lib/buscampz_rayito_flutter_package/example/example_screens/client_answer_card_screen.dart
#	lib/buscampz_rayito_flutter_package/example/example_screens/incidents_text_field_screen.dart
#	lib/buscampz_rayito_flutter_package/example/example_screens/textfield_with_button_send_screen.dart
#	lib/buscampz_rayito_flutter_package/example_screen/example_screens/app_bar_example_screen.dart
#	lib/buscampz_rayito_flutter_package/example_screen/example_screens/bottom_sheet_example_screen.dart
#	lib/buscampz_rayito_flutter_package/example_screen/example_screens/client_answer_card_example_screen.dart
#	lib/buscampz_rayito_flutter_package/example_screen/example_screens/client_answer_card_screen.dart
#	lib/buscampz_rayito_flutter_package/example_screen/example_screens/detail_incident_text_field_example_screen.dart
#	lib/buscampz_rayito_flutter_package/example_screen/example_screens/detail_incident_text_field_screen.dart
#	lib/buscampz_rayito_flutter_package/example_screen/example_screens/evidence_elevated_button_example_screen.dart
#	lib/buscampz_rayito_flutter_package/example_screen/example_screens/evidence_elevated_button_screen.dart
#	lib/buscampz_rayito_flutter_package/example_screen/example_screens/example_screens_export.dart
#	lib/buscampz_rayito_flutter_package/example_screen/example_screens/incidence_section_example_screen.dart
#	lib/buscampz_rayito_flutter_package/example_screen/example_screens/incidents_text_field_example_screen.dart
#	lib/buscampz_rayito_flutter_package/example_screen/example_screens/incidents_text_field_screen.dart
#	lib/buscampz_rayito_flutter_package/example_screen/example_screens/textfield_with_button_send_example_screen.dart
#	lib/buscampz_rayito_flutter_package/example_screen/example_screens/textfield_with_button_send_screen.dart
